### ISSUE TITLE
Pidof command not found on macOS Github runners

### ISSUE THREAD

**Issue by dela-gt** [kielabokkie/ssh-key-and-known-hosts-action#31]
```bash
Post job cleanup.
Stopping ssh-agent
/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1/dist/cleanup/index.js:1368
		error = new Error(message);
		        ^
Error: Command failed with exit code 1: kill -9 $(pidof ssh-agent)
/bin/sh: pidof: command not found
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
    at makeError (/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1/dist/cleanup/index.js:1368:11)
    at handlePromise (/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1/dist/cleanup/index.js:[2](https://github.com/groovetime/groovetime-app/actions/runs/8007973882/job/21873313014#step:17:2)722:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1/dist/cleanup/index.js:645:[3](https://github.com/groovetime/groovetime-app/actions/runs/8007973882/job/21873313014#step:17:3)) {
  shortMessage: 'Command failed with exit code 1: kill -9 $(pidof ssh-agent)',
  command: 'kill -9 $(pidof ssh-agent)',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '/bin/sh: pidof: command not found\n' +
    'kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
Node.js v20.8.1
```

This was fixed in #24 but reintroduced after version `1.4.0` release. 

I can see the change in the `dist` directory here for [1.4.0](https://github.com/kielabokkie/ssh-key-and-known-hosts-action/blob/master/dist/cleanup/index.js#L645) vs [1.3.5](https://github.com/kielabokkie/ssh-key-and-known-hosts-action/blob/v1.3.5/dist/cleanup/index.js#L645)

**Comment by kielabokkie** [comment]
Thanks for reporting @dela-gt. This should be fixed in version v1.4.1. Let me know if you still encounter any problems.

**Comment by dela-gt** [comment]
Thank you @kielabokkie for the quick fix. It's working 🎉
